people/students/laurenz.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Laurenz Kremeyer
+position: Ph.D. Student
+email: alice.bob@mail.mcgill.ca
```

Comment:
Probably not your real e-mail

Reply:
Oh, thanks for letting me know. Can I change it in this PR, or do I need to open another one?

Reply:
You can change it in this PR by: 
1. making the change on your computer;
2. committing that change to your local master branch (`git commit -am "your message here"`)
3. pushing. The change should appear here (`git push`)

Reply:
Thanks!